clang/test/dpct/disable-all-extensions.cu

```diff
@@ -402,6 +402,16 @@ static void func() {
   // CHECK-NEXT: MY_ERROR_CHECKER(0);
   MY_ERROR_CHECKER(cudaStreamQuery(s0));
 
+  // CHECK: /*
+  // CHECK-NEXT: DPCT1026:{{[0-9]+}}: The call to cuStreamQuery was removed because SYCL currently does not support query operations on queues.
+  // CHECK-NEXT: */
+  cuStreamQuery(s0);
```

Comment:
Could you give an example of how this API is used in the real code?

The return value of APIs like `cu[da]EventQuery/cu[da]StreamQuery` is a little different from other APIs.
It can return a non-zero value (such as `CUDA_ERROR_NOT_READY`) which is not an error.
E.g., for cudaEventQuery in below code we cannot let the migrated API always return 0.
```
    unsigned long int counter = 0;
    while (cudaEventQuery(stop) == cudaErrorNotReady) {
        counter++;
    }
```

Reply:
Here is the snippent of a code from an project.

```cuda
	CUresult result = cuStreamQuery((CUstream)hStream );

	RSAssert( result==CUDA_SUCCESS || result==CUDA_ERROR_NOT_READY, "Failed to query stream");
```

Reply:
I'm OK to replace this API with 0 if it is not "really" used in the project like your example.
If you want to support the in-loop scenario, you can refer to PR https://github.com/oneapi-src/SYCLomatic/pull/1970.